### DISCUSSION TITLE
How to add exchange rate?

### DISCUSSION THREAD

**Discussion by AlexHladin** [enrique-lozano/Monekin#160] [Q&A] [answered]
After I created an account with the non-standard currency I saw a warning that the exchange rate is 1. How can I change it or connect to some currency exchange API?

**Comment by enrique-lozano** [comment, marked as answer]
Hi! First of all, thank you for using the app. It's a pleasure to see that this is growing fast.

To create a new exchange rate, go to the currency manager:

![image](https://github.com/enrique-lozano/Monekin/assets/61509169/04c993f1-ddc5-4c5a-9c84-52f71f3ab494)

Then you can create your exchange rate by pressing the "Add" button and filling the form:

![image](https://github.com/enrique-lozano/Monekin/assets/61509169/9b010052-26cb-4a4e-97bc-b063bb1ca7de)

Hope it helps!

**Reply by AlexHladin** [reply]
Thanks, your answer helps a lot!
Any chance that the app will have automatic currency exchange synchronization through some public API?

**Reply by enrique-lozano** [reply]
Hello! It is not currently possible but this possibility has been on my mind for a long time. It's a pretty big feature to add and now I prefer to spend time on smaller features and bug fixes. It will definitely be one of the functionalities to be implemented if the application grows at a good pace.

In any case, integrating exchange rates with an API would be optional, an extra to the process of manually adding/editing exchange rates. It would be necessary to study how to implement this in terms of UX/UI.

Of course I accept any PR in this sense.
